src/network/ForwardProxy.ts

```diff
@@ -299,7 +300,9 @@ class ForwardProxy {
       return;
     }
     const url = new URL(`pk://${request.url}`);
-    const nodeId = url.searchParams.get('nodeId') as NodeId | null;
+    const rawNodeId = url.searchParams.get('nodeId');
+    const nodeId =
+      rawNodeId != null ? nodesUtils.decodeNodeId(rawNodeId) : undefined;
```

Comment:
Replace this with just:

```ts 
const nodeIdEncodedForURL = url.searchParams.get('nodeId');
const nodeId = nodeIdEncodedForURL != null ? nodesUtils.decodeNodeId(nodeIdEncodedForURL) : undefined;
```

Use the same name as when we encoded as well. Keeps it tidy and searchable.

You just need to change `decodeNodeId` to take `any` type and return `undefined` if it isn't a string.

Reply:
Disregard:

~You just need to change `decodeNodeId` to take `any` type and return `undefined` if it isn't a string.~

Reply:
Fixed.

Reply:
Ah, i'll roll that one change back.

Reply:
Yep, just change these 3 lines accordingly.